lib/widgets/subscription_list.dart

```diff
@@ -77,10 +78,8 @@ class _SubscriptionListPageBodyState extends State<SubscriptionListPageBody> wit
 
     // TODO: Implement collapsible topics
 
-    // TODO(i18n): localize strings on page
-    //   Strings here left unlocalized as they likely will not
-    //   exist in the settled design.
```

Comment:
Huh, what was this about?

Reply:
This was from #397, which followed [an old design](https://www.figma.com/design/1JTNtYo9memgW7vV6d0ygq/Zulip-Mobile?node-id=171-12359&t=ilEg6ymsxO6JLNQ5-0). The comment is no longer accurate as we localize the strings in the PR. It doesn't seem useful to revise this, either.